src/components/Calculator.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable */
```

Comment:
- [ ] Please kindly do not disable linters, fix linters manually helps programmers to write clean and clear codebase 😁 😉